<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferDialog</class>
 <widget class="QWidget" name="TransferDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Перевод средств</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="toLabel">
     <property name="text">
      <string>Получатель (ID либо IBAN)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="toEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="amountLabel">
     <property name="text">
      <string>Сумма</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="amountEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Описание</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="descriptionEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string>Информация о переводе</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="infoEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="beforeTrInfoButton">
       <property name="text">
        <string>Предварительная информация</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>Отправить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
